src/base/base.scss

```diff
@@ -77,10 +77,63 @@ button {
 }
 
 details {
-  summary { cursor: pointer; }
+  summary {
+    cursor: pointer;
+  }
 
   &:not([open]) {
     // Set details content hidden by default for browsers that don't do this
-    > *:not(summary) { display: none !important; }
+    > *:not(summary) {
+      display: none !important;
+    }
+  }
+}
+
+// global focus styles
+
+a,
+button,
+[role='button'],
+input,
+textarea,
+select {
+  // fallback :focus state
+  &:focus {
+    @include focusOutline;
+
+    // remove fallback :focus if :focus-visible is supported
+    &:not(:focus-visible) {
+      outline: solid 1px transparent;
+    }
+  }
+
+  // default focus state
+  &:focus-visible {
+    @include focusOutline;
+  }
+}
+
+input:not([type='radio'], [type='checkbox']),
+textarea,
+select {
+  &:focus,
+  &:focus-visible {
+    outline-offset: 0;
+  }
+}
+
+// for handling focus conditionally
+.focus {
+  @include focusOutline;
+
+  &:is(input, textarea, select) {
+    outline-offset: 0;
+  }
+}
+
+// Windows High Contrast mode
+@media (forced-colors: active) {
```

Comment:
Nice - I've never used the `forced-colors` media query.

Reply:
I'm hoping someone with a VM from the accessibility team can test this! ☺️

Reply:
Hello from the future 😄 

Was someone able to confirm that this works?